superset/views/base.py

```diff
@@ -170,16 +170,61 @@ def json_response(self, obj, status=200):
             mimetype="application/json",
         )
 
+    def menu_data(self):
+        menu = appbuilder.menu.get_data()
+        root_path = "#"
+        if not g.user.is_anonymous:
+            try:
+                logo_target_path = appbuilder.app.config.get(
+                    "LOGO_TARGET_PATH"
+                ) or "/profile/{}/".format(g.user.username)
+                if logo_target_path.startswith("/"):
+                    root_path = "/superset{}".format(logo_target_path)
+                else:
+                    root_path = logo_target_path
+            except NameError as e:
+                logging.exception(e)
+
+        languages = {}
+        for lang in appbuilder.languages:
+            languages[lang] = {
+                **appbuilder.languages[lang],
+                "url": appbuilder.get_url_for_locale(lang),
+            }
+        return {
+            "menu": menu,
+            "brand": {
+                "path": root_path,
+                "icon": appbuilder.app_icon,
+                "alt": appbuilder.app_name,
+            },
+            "navbar_right": {
+                "bug_report_url": appbuilder.app.config.get("BUG_REPORT_URL"),
+                "documentation_url": appbuilder.app.config.get("DOCUMENTATION_URL"),
+                "languages": languages,
+                "show_language_picker": appbuilder.app.config.get(
+                    "SHOW_TRANSLATION_PICKER", True
+                ),
+                "user_is_anonymous": g.user.is_anonymous,
+                "user_info_url": appbuilder.get_url_for_userinfo,
```

Comment:
Weird to see a `get_` method that's not a call... @dpgaspar ?

Reply:
Agree it's weird, I'm planning to change these appbuilder properties on FAB 3, since this would force a breaking change.